src/main/java/frc/robot/constants/Constants.java

```diff
@@ -0,0 +1,30 @@
+package frc.robot.constants;
+
+import frc.robot.drive.MotorType;
+
+public interface Constants {
```

Comment:
Should add a class comment describing the purpose of this class. The comment can also talk about what to do when creating a new robot constants file.